File: TGPR.Database/TGPR.Database.Common/Models/Applications/ApplicationCategoryModel.cs
using System.Collections.Generic;

namespace TGPR.Database.Common.Models.Applications
{
    public class ApplicationCategoryModel
    {
        public int ApplicationCategoryId { get; set; }
        public int ApplicationTemplateId { get; set; }
        public string Name { get; set; }
        public int ApplicationSortOrder { get; set; }
        public int ReviewerSortOrder { get; set; }
        public bool Deleted { get; set; }
        public bool HasReview { get; set; }

        public IEnumerable<ApplicationQuestionModel> Questions { get; set; }
    }
}

File: TGPR.Database/TGPR.Database.Api/client/src/app/_models/applications/application-category.model.ts
import {ApplicationQuestionModel} from "./application-question.model";
import {ApplicationCategoryReviewModel} from "./application-category-review.model";

export class ApplicationCategoryModel {
    ApplicationCategoryId: number;
    ApplicationTemplateId: number;
    Name: string;
    ApplicationSortOrder: number;
    ReviewerSortOrder: number;
    Deleted: boolean;
    HasReview: boolean;

    Questions: ApplicationQuestionModel[];

    Review: ApplicationCategoryReviewModel;
}